srcpkgs/msbuild/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'msbuild'
+pkgname=msbuild
```

Comment:
since the package is not build from source, `pkgname` should be suffixed with `-bin`.

Reply:
Ah ok, fixed